src/sentry/features/__init__.py

```diff
@@ -204,6 +204,7 @@
 default_manager.add("organizations:team-insights", OrganizationFeature)
 default_manager.add("organizations:derive-code-mappings", OrganizationFeature)
 default_manager.add("organizations:derive-code-mappings-dry-run", OrganizationFeature)
+default_manager.add("organizations:codecov-stacktrace-integration", OrganizationFeature, True)
```

Comment:
What does the True value imply? It makes me think that the feature is on by default.

Reply:
That boolean parameter is for `entity_feature` which indicates that the flag can be managed in Flagr.